src/renderer/components/ft-toast/ft-toast.vue

```diff
@@ -0,0 +1,18 @@
+<template>
+  <div class="toast-holder">
+    <div
+      v-for="(toast, index) in toasts"
+      :key="'toast-' + index"
+      class="toast"
+      :class="{ closed: !toast.isOpen, open: toast.isOpen }"
+      @click="performAction(toast)"
```

Comment:
Let's pass the index instead of the object itself.  This will make the other logic simpler to deal with.